Makefile

```diff
@@ -1,4 +1,34 @@
-.PHONY: fmt-check lint fmt
+prefix = /usr
+
+all:
+	: # do nothing
+
+build:
+
+	cd mgr && shards install && shards build
+
+install: build
+
+	install -D mgr/bin/kadalu \
+                $(DESTDIR)$(prefix)/sbin/kadalu
+	install -d clients/python/kadalu \
```

Comment:
Path is `clients/python/kadalu_storage`. Also after this PR merge: `clients` is renamed to `sdk`. Path: `sdk/python/kadalu_storage`.

https://github.com/kadalu/moana/pull/216

Reply:
changed